addons/l10n_ec/data/account.account.template.csv

```diff
@@ -114,6 +118,9 @@ ec21050102,21050102,Otras provisiones locales,TRUE,liability_current,l10n_ec_ifr
 ec210502,210502,Provisiones del exterior,TRUE,liability_current,l10n_ec_ifrs
 ec2106,2106,Porcion corriente de obligaciones emitidas,FALSE,liability_current,l10n_ec_ifrs
 ec21070101,21070101,Impuesto a la renta retenido a empleados,TRUE,liability_current,l10n_ec_ifrs
+ec21070102,21070102,Impuesto por pagar IVA,TRUE,liability_current,l10n_ec_ifrs
+ec21070103,21070103,Retenciones en la fuente por pagar,TRUE,liability_current,l10n_ec_ifrs
+ec21070104,21070104,Impuesto ICE por pagar,TRUE,liability_current,l10n_ec_ifrs
```

Comment:
Maybe you need to add an "others" account?

Reply:
@stevTresCloud  Add missing account "Otros impuestos por pagar"